### DISCUSSION TITLE
Thank you, gracias, grazie, danke...

### DISCUSSION THREAD

**Discussion by caouecs** [Laravel-Lang/common#6] [Q&A]
| Lang        | Thank you                                            | You're welcome                                             |
|-------------|:-----------------------------------------------------|:-----------------------------------------------------------|
| ar          | شكراً / شكراً لك                                     | العفو / لا شكر على واجب                                    |
| az          | Təşəkkür edirəm                                      | Buyurun / Xoşdur                                           |
| be          | Дзякуй                                               | Калі ласка                                                 |
| bg          | Благодаря ти                                         | Моля                                                       |
| bn          | তোমাকে ধন্যবাদ / আপনাকে ধন্যবাদ                      | তোমাকে স্বাগতম / আপনাকে স্বাগতম                            |
| bs          | Hvala                                                | Nema na čemu                                               |
| ca          | Gràcies                                              | De res                                                     |
| cs          | Dík                                                  | Není zač / Prosím                                          |
| cy          | Diolch                                               | Croeso                                                     |
| da          | Tak                                                  | Selv tak / Det var så lidt                                 |
| de          | Danke                                                | Bitte / Bitteschön                                         |
| de_CH       | Danke                                                | Bitte / Bitteschön                                         |
| el          | Ευχαριστούμε                                         | Παρακαλώ                                                   |
| es          | Gracias / Muchas gracias                             | De nada                                                    |
| et          | Aitäh                                                | Pole tänu väärt / Palun                                    |
| eu          | Eskerrik asko                                        | Ez horregatik                                              |
| fa          | ممنون                                                | خواهش می‌کنم                                               |
| fi          | Kiitos                                               | Ole hyvä / Ei kestä                                        |
| fil         | Salamat                                              | Walang anuman                                              |
| fr          | Merci                                                | De rien / Je vous en prie                                  |
| gl          | Grazas                                               | De nada                                                    |
| he          | תודה                                                 | בבקשה                                                      |
| hi          | धन्यवाद                                              | आपका स्वागत है                                             |
| hr          | Hvala                                                | Nema na čemu                                               |
| hu          | Köszönöm                                             | Szívesen / Nincs mit                                       |
| hy          | Շնորհակալություն                                     | Խնդրեմ                                                     |
| id          | Terima kasih                                         | Sama sama / Terima kasih kembali                           |
| is          | Takk                                                 | Gerðu svo vel / Ekkert mál                                 |
| it          | Grazie                                               | Prego                                                      |
| ja          | ありがとうございます                                           | どういたしまして                                                   |
| ka          | გმადლობთ                                             | არაფრის                                                    |
| kk          | Рақмет                                               | Оқасы жоқ / Әрине                                          |
| km          | សូមអរគុណ                                             | មិនអីទេ                                                    |
| kn          | ಧನ್ಯವಾದಗಳು                                           | ಸ್ವಾಗತ                                                     |
| ko          | 감사합니다                                                | 천만에요 / 아니에요                                                |
| lt          | Ačiū                                                 | Prašom / Nėra už ką                                        |
| lv          | Paldies                                              | Lūdzu / Nav par ko                                         |
| me          | Hvala                                                | Nema na čemu                                               |
| mk          | Ви благодарам                                        | Нема на што / Повелете                                     |
| mn          | Баярлалаа                                            | Зүгээр дээ                                                 |
| mr          | धन्यवाद                                              | तुमचे स्वागत आहे                                           |
| ms          | Terima kasih                                         | Sama-sama                                                  |
| nb          | Takk                                                 | Bare hyggelig / Vær så god                                 |
| ne          | धन्यवाद                                              | तपाईंलाई स्वागत छ                                          |
| nl          | Dankjewel                                            | Graag gedaan                                               |
| nn          | Takk                                                 | Berre hyggjeleg / Ver så god                               |
| pl          | Dziękuję                                             | Proszę bardzo                                              |
| ps          | مننه                                                 | هېڅ خبره نشته                                              |
| pt          | Obrigado / Obrigada                                  | De nada                                                    |
| pt_BR       | Obrigado / Obrigada                                  | De nada / Por nada / Não tem de quê                        |
| ro          | Mulțumim                                             | Cu Placere                                                 |
| ru          | Спасибо                                              | Пожалуйста / Всегда пожалуйста / Обращайтесь / Без проблем |
| sc          | Gratzias                                             | De nudda                                                   |
| si          | ස්තුතියි                                             | කමක් නැහැ                                                  |
| sk          | Hvala vam                                            |                                                            |
| sl          | Hvala                                                | Prosim                                                     |
| sq          | Faleminderit                                         | S'ka përse / Ju lutem                                      |
| sr_Cyrillic | Хвала                                                | Нема на чему                                               |
| sr_Latin    | Hvala                                                | Nema na čemu                                               |
| sv          | Tack                                                 | Varsågod / Ingen fara                                      |
| sw          | Ahsante                                              |                                                            |
| tg          | Ташаккур                                             | Марҳамат / Ҳеҷ гап не                                      |
| th          | ขอบคุณ                                               | ยินดีครับ/ค่ะ                                              |
| tk          | Sag boluň                                            | Arman ýok                                                  |
| tl          | Salamat                                              | Walang anuman                                              |
| tr          | Teşekkür Ederim / Teşekkür Ederiz (much more formal) | Rica ederim                                                |
| ug          | رەھمەت                                               | ئەرزىمىدۇ                                                  |
| uk          | Дякую / Дуже дякую                                   | Будь ласка                                                 |
| ur          | شکریہ                                                | کوئی بات نہیں                                              |
| uz_Cyrillic | Рахмат                                               | Арзимайди                                                  |
| uz_Latin    | Rahmat                                               | Arzimaydi                                                  |
| vi          | Cảm ơn                                               | Không có gì                                                |
| zh_CN       | 谢谢                                                   | 不客气                                                        |
| zh_HK       | 多謝                                                   | 唔使客氣                                                       |
| zh_TW       | 感謝                                                   | 不客氣                                                        |

**Comment by olsza** [comment]
PL:
thanks = dzięki
thank you = dziękuję
:)

**Comment by andrey-helldar** [comment]
| Language | key | value |
|:---:|:---|:---|
| RU | Thanks | Спасибо |
| RU | Thank you | Спасибо / Спасибо Вам |
| DE | Thanks | Danke / Dankeschön |
| RU | Many thanks | Большое спасибо |
| DE | Many thanks | Vielen dank |

**Comment by mehmetnky** [comment]
[tr]
Thanks = Teşekkürler
Thank You = Teşekkür Ederim / Teşekkür Ederiz (much more formal)

**Reply by andrey-helldar** [reply]
Teşekkür Ederim 🙂

**Comment by mohamedsabil83** [comment]
key | value
-----|---------
Thanks | شكراً
Thank you | شكراً / شكراً لك
Many Thanks | عظيم الشكر

**Reply by andrey-helldar** [reply]
شكراً لك

**Comment by arman-arif** [comment]
| en                   | bn                                                    |
| -------------------|------------------------------------------------|
| Thanks           | ধন্যবাদ                                              |
| Thank you      | তোমাকে ধন্যবাদ / আপনাকে ধন্যবাদ     |
| Many Thanks | অনেক ধন্যবাদ                                    |

**Reply by andrey-helldar** [reply]
তোমাকে ধন্যবাদ

**Reply by mohamedsabil83** [reply]
তোমাকে ধন্যবাদ

**Reply by arman-arif** [reply]
| en | bn |
| -------------------|--------------------------|
| Welcome | স্বাগতম |
| You're Welcome | তোমাকে স্বাগতম / আপনাকে স্বাগতম ।

**Reply by andrey-helldar** [reply]
তোমাকে ধন্যবাদ

**Comment by BellezaEmporium** [comment]
FR : 
You're welcome : De rien / Je vous en prie.

**Reply by andrey-helldar** [reply]
Merci

**Reply by caouecs** [reply]
c'est " de rien " ;)

**Reply by BellezaEmporium** [reply]
Effectivement, dépendant de la personne, les deux se valent, même si la version officielle veut que "de" et "rien" soient détachés.

**Reply by caouecs** [reply]
absolument pas du tout, "derien" n'existe pas en français

**Reply by BellezaEmporium** [reply]
> absolument pas du tout, "derien" n'existe pas en français

Effectivement, ça sera à modifier. On peut aussi ajouter "avec plaisir", qui est une variante de "you're welcome", sous la forme "with pleasure".

**Comment by EuCarlos** [comment]
**pt_BR**

Thanks: Brigado / Brigada (informal)
Thank you: Obrigado / Obrigada
Many Thanks: Muito Obrigado

You're Welcome: De nada / Por nada / Não tem de quê

**Reply by andrey-helldar** [reply]
Obrigado

**Reply by EuCarlos** [reply]
De nada 👍🏾

**Comment by wadakatu** [comment]
| en | ja |
| ------------- | ------------- |
| You're Welcome. | どういたしまして  |

Hope it helps.

**Reply by andrey-helldar** [reply]
ありがとうございます

**Comment by aurawindsurfing** [comment]
Awesome package guys! All that is missing is some flags and maybe language switcher ;-)

| Lang | Thank you | You're welcome  |
| ------- | --- | --- |
| pl | Dziękuję | Proszę bardzo |

**Comment by k1rana** [comment]
| en             | id                               | 
|----------------|----------------------------------|
| Thank you      | Terima kasih                     | 
| You're welcome | Sama sama / Terima kasih kembali |

**Reply by andrey-helldar** [reply]
Terima kasih

**Reply by k1rana** [reply]
Sama-sama.
Btw, this discussion should be a formal one, right? If it's a formal “Sama-sama”, and informally “Sama sama”.

**Reply by andrey-helldar** [reply]
In this case, both options will work.
We use these phrases when conducting code reviews.
